app/views/moderator_applications/show.html.slim

```diff
@@ -0,0 +1,38 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.restore-defaults.col-lg-8
+      h1
+        = link_to(@application.user.username, user_path(@application.user))
+        | 's Moderator Application
+
+      - if current_user.is_admin?
+        a.btn.btn-primary href=moderator_applications_path style='margin-right: 5px;'
+          span.fa.fa-undo Back
+      = link_to '<span class="fa fa-edit"></span>'.html_safe, edit_moderator_application_path(@application), class: 'btn btn-primary', style: 'margin-right: 5px;'
+      = link_to '<span class="fa fa-trash"></span>'.html_safe, @application, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-primary'
+
+      h2 What time zone are you in?
```

Comment:
This Q/A code is repetitive and should probably be factored out.

Reply:
The questions in particular are hard-coded between `show` and `edit` functions and probably should be moved.

Also, I was considering using a `dl` for this stuff in the *form*, but it doesn't feel like it would make sense here. Or maybe it would? I'll have to think about it more, but I should probably try to keep it consistent between them.

Reply:
Regarding a `dl` on the form, I think the way it would look would be something like:
```html
dl
  ...

  div
    dt = form.label(:whatever, 'What periods of time can you be actively moderating the site?')
    dt
      small Please answer in Central Standard Time.
    dd = form.textarea(:whatever)

  div
    ...

  ...
```
And style the elements semantically with a `form > dl` selector instead of with the `col` tags.